json/supermarches.js

```diff
@@ -657,17 +657,17 @@ export const supermarches = [
   },
 
   {
-    nom: 'La source',
+    nom: 'La Source',
     departement: '75',
     ville: 'Paris XXème',
-    adresse: 'rue sorbier',
+    adresse: '4 rue Félix Terrier, 75020 Paris',
     logo: 'la_source.png',
     site_web: 'http://cooplasource.fr/',
     courriel: 'contact@cooplasource.fr',
     facebook: 'https://www.facebook.com/cooplasource/',
     avancement: 3,
```

Comment:
Ah, je viens de voir que nous serions pluôt en avancement 4.
```suggestion
    avancement: 4,
```